volume-providers/vpc/vpcclient/models/volume_attachment.go

```diff
@@ -23,14 +23,14 @@ type Device struct {
 
 // VolumeAttachment for riaas client
 type VolumeAttachment struct {
-	ID   string `json:"id"`
+	ID   string `json:"id,omitempty"`
 	Href string `json:"href,omitempty"`
 	Name string `json:"name,omitempty"`
 	// Status of volume attachment named - attaching , attached, detaching
 	Status string `json:"status,omitempty"`
 	Type   string `json:"type,omitempty"` //boot, data
 	// InstanceID this volume is attached to
-	InstanceID *string    `json:"instance_id,omitempty"`
+	InstanceID *string    `json:"-"`
```

Comment:
why do not want to trace this, I think it would be good if we can trace this value as well

Reply:
As "VolumeAttachment" Object is used both as an INPUT PARAM during attach and also as a Response Object.
But all the param are now not accepted as valid during INPUT, hence being skipped during the requestObject generation. VPC-GC used to accept unnecessary payloads also ...but VPC-NG rejects/errors out , when extra payload params are specified.